src/System.Linq.Expressions/tests/ExceptionHandling/ExceptionHandlingExpressions.cs

```diff
@@ -288,93 +282,60 @@ public void NonExceptionDerivedExceptionWrappedInterpreted()
             Assert.Equal("System.Runtime.CompilerServices.RuntimeWrappedException", ex.GetType().FullName);
```

Comment:
I don't have any context on RuntimeWrappedException... I think this is the first time I've heard of it.